<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TangQTTestClass</class>
 <widget class="QMainWindow" name="TangQTTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TangQTTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ASTSolver</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_2">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ASTStringer</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit_3">
          <property name="styleSheet">
           <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>source code:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>result:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Consolas&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
  </widget>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="TangQTTest.qrc">
     <normaloff>:/new/prefix1/EE0D258040D18DB8558AE92A70FAFACF.jpg</normaloff>:/new/prefix1/EE0D258040D18DB8558AE92A70FAFACF.jpg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TangQTTest.qrc"/>
 </resources>
 <connections/>
</ui>
